js/core/src/Blocks.svelte

```diff
@@ -227,13 +228,23 @@
 		return get_data(component_id);
 	}
 
+	function check_inputs_ready(): boolean {
```

Comment:
Nit: we don't need a function for this, can just use `inputs_waiting.length > 0` in the code directly